proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -201,32 +201,44 @@ message TimeIntervalFilter {
 
 // Parameters for filtering the dispatch list
 message DispatchFilter {
+  // Recurrence filters
+  //
+  // The fields are left commented out for now as the exact definition and requirements
+  // for recurrence filtering are still being finalized.
   message RecurrenceFilter {
-    // Filter by recurring status
-    // True: Only recurring dispatches will be returned.
-    // False: Only non-recurring dispatches will be returned.
-    // Unset: Both recurring and non-recurring dispatches will be returned,
-    //        recurring dispatches will be filtered based on the recurrence
-    //        filter, while non-recurring dispatches will be returned regardless.
-    // Examples:
-    // - To retrieve only recurring dispatches:
-    //   filter { recurrence_filter { is_recurring: true } }
-    // - To retrieve only non-recurring dispatches:
-    //   filter { recurrence_filter { is_recurring: false } }
-    // - To retrieve all dispatches:
-    //   filter { recurrence_filter {} }
-    // - To retrieve only recurring dispatches with a specific frequency:
-    //   filter { recurrence_filter { is_recurring: true, freq: [FREQUENCY_DAILY] } }
-    // - To retrieve only recurring dispatches with a specific frequency but
-    //   also include non-recurring dispatches:
-    //   filter { recurrence_filter { freq: [FREQUENCY_DAILY] } }
-
-    // - only_recurring = true: Only recurring dispatches will be returned.
-    optional bool is_recurring = 1;
-
-    // Filter by frequency
-    // If not specified, all frequencies will be returned.
-    repeated RecurrenceRule.Frequency freqs = 2;
+    // The frequency specifier of this recurring dispatch
+    //optional RecurrenceRule.Frequency freq = 1;
+
+    //// How often this dispatch should recur, based on the frequency
+    //// Example:
+    //// - Every 2 hours:
+    ////   freq = FREQUENCY_HOURLY
+    ////   interval = 2
+    //// Valid values typically range between 1 and 10_000, depending on the implementation.
+    //optional uint32 interval = 2;
```

Comment:
Should these fields really be commented out?

Reply:
We didn't discuss anywhere yet how exactly recurrence filtering should be done and I'd rather leave it undefined and add it once discussed (and possibly implemented) than adding something now that might be breaking to clients later.